data/mods/innawood/overmap/overmap_terrarin/overmap_terrain_stream.json

```
@@ -0,0 +1,26 @@
+[
+  {
+    "type": "overmap_terrain",
+    "id": "stream",
+    "copy-from": "generic_open_land",
+    "name": "stream",
+    "sym": ".",
+    "color": "light_blue"
+  },
+  {
+    "type": "overmap_terrain",
+    "id": "stream_end",
+    "copy-from": "generic_open_land",
+    "name": "stream",
+    "sym": ".",
+    "color": "light_blue"
+  },
+  {
+    "type": "overmap_terrain",
+    "id": "stream_corner",
+    "copy-from": "generic_open_land",
+    "name": "stream",
+    "sym": ".",
+    "color": "light_blue"
+  }
+]
```

```suggestion
[
  {
    "type": "overmap_terrain",
    "id": [ "stream", "stream_end", "stream_corner" ],
    "copy-from": "generic_open_land",
    "name": "stream",
    "sym": ".",
    "color": "light_blue"
  }
]
```
Am I missing some reason why it's necessary to split these up?

---

No, you aren't. I did not realise that the IDs could be put into a list like that. I'll do this in the future.